hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -1291,15 +1296,18 @@ protected static List<HoodieColumnRangeMetadata<Comparable>> getLogFileColumnRan
           .withStorage(datasetMetaClient.getStorage())
           .withBasePath(datasetMetaClient.getBasePath())
           .withLogFilePaths(Collections.singletonList(filePath))
-          .withBufferSize(MAX_DFS_STREAM_BUFFER_SIZE.defaultValue())
+          .withBufferSize(maxBufferSize)
```

Comment:
Can we pass around partition path + file name instead? The `concatenation` should be more fast than `subString/lastIndexOf`.